library/manageiq_user_management.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+
+import os
+from ansible.module_utils.basic import *
```

Comment:
I like to move the ansible bits under 

`if __name__`

section.  This allows me to import your code in python and work with it.  Consider moving it.